### ISSUE TITLE
SonarrPlex don't display Images 

### ISSUE THREAD

**Issue by rickytin** [danbovey/SonarrPlex#9]
I install the extension and  it docent display the Images.

**Comment by enchained** [comment]
Can confirm the problem. Image loading returns 401 Unauthorized.
Image URL is formed like that:
http://plexURL/sonarrURL/MediaCover/17/poster-250.jpg?lastWrite=636425094770000000
instead of just
http://sonarrURL/MediaCover/17/poster-250.jpg?lastWrite=636425094770000000

Images work normal with app.plex.tv, but not on custom ip/domain.

@danbovey, could you please look into this?

**Comment by danbovey** [comment]
Yep, I'm getting the same thing. Note the single slash after http which would cause the browser to think it's a relative link.

```
http:/plexIP/sonarrURL/MediaCover/9/poster-250.jpg?lastWrite=636512294360000000
```

I'll look into this code ASAP: [js/poster.js#L88](https://github.com/danbovey/SonarrPlex/blob/master/js/poster.js#L88)

**Comment by danbovey** [comment]
Lol I was using `path.join`, not `url.resolve`.

**Comment by danbovey** [comment]
Fix published in version `1.0.6`.
